packages/ui/src/components/va-time-input/VaTimeInput.vue

```diff
@@ -36,32 +57,42 @@ import VaTimePicker from '../va-time-picker/VaTimePicker.vue'
 import VaInput from '../va-input/VaInput.vue'
 import { useSyncProp } from '../../composables/useSyncProp'
 import { useValidation, useValidationProps, useValidationEmits } from '../../composables/useValidation'
+import { useClearableProps, useClearable } from '../../composables/useClearable'
 import { useTimeParser } from './hooks/time-text-parser'
 import { useTimeFormatter } from './hooks/time-text-formatter'
 import { extractComponentProps, filterComponentProps } from '../../utils/child-props'
 
+const propsExceptions = ['rules', 'error', 'errorMessages', 'clearable', 'clearableIcon']
```

Comment:
```suggestion
const vaInputProps = extractComponentProps(VaInput, ['rules', 'error', 'errorMessages', 'clearable', 'clearableIcon'])
```

`propsExceptions` not clear name

Reply:
fixed it